src/main/java/seedu/address/logic/commands/AddLeaveCommand.java

```diff
@@ -54,6 +54,9 @@ public class AddLeaveCommand extends Command {
      * Creates an AddLeaveCommand to add the specified {@code Leave}
      */
     public AddLeaveCommand(Index index, Title title, Range dates, Description description) {
+        requireNonNull(index);
+        requireNonNull(title);
+        requireNonNull(dates);
```

Comment:
I think model.hasLeave() in execute() can be replaced with model.hasConcurrentLeave()

Reply:
ummm so do we still do the concurrent leave, or i just revert my recent commit?